dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -65,7 +65,7 @@
 Name:           {{ name }}
 Summary:        {{ summary|default("Empty") }}
 Version:        {{ version|default("1") }}
-Release:        {{ release|default("1") }}%{?dist}
+Release:        {{ release|default("1") }}
```

Comment:
I am afraid that removing the dist tag could break some existing tests (I didn't try it yet though).
Do you think that preserving it and doing the actual comparison as follows is a bad idea?
`(release is None or p.release == release or p.release.split('.')[0] == release):`